src/Rules/StarterWeb/AI/OrganizationalAuth/Multiple/Startup.cs

```diff
@@ -75,16 +77,13 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
 
             app.UseStaticFiles();
 
-            app.UseCookieAuthentication(new CookieAuthenticationOptions
-            {
-                AutomaticAuthenticate = true
-            });
+            app.UseCookieAuthentication();
 
             app.UseOpenIdConnectAuthentication(new OpenIdConnectOptions
             {
                 ClientId = Configuration["Authentication:AzureAd:ClientId"],
                 Authority = Configuration["Authentication:AzureAd:AADInstance"] + "Common",
-                SignInScheme = CookieAuthenticationDefaults.AuthenticationScheme,
+                CallbackPath = Configuration["Authentication:AzureAd:CallbackPath"],
```

Comment:
it should be `/signin-oidc` https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectOptions.cs#L52